showcase/app/templates/page-components/advanced-table.hbs

```diff
@@ -738,9 +738,52 @@
 
   <Shw::Divider />
 
-  <Shw::Text::H3>Functional examples</Shw::Text::H3>
+  <Shw::Text::H2>Pinnable first column</Shw::Text::H2>
 
-  <Shw::Text::H4>With dynamic focusable content in cells</Shw::Text::H4>
+  <Shw::Grid @columns={{2}} @gap="2rem" as |SG|>
+    {{#let (array false true) as |selectableBools|}}
+      {{#each selectableBools as |selectableBool|}}
+        {{#let (array false true) as |sortableBools|}}
+          {{#each sortableBools as |sortableBool|}}
+            <SG.Item @label="isSelectable={{selectableBool}}, isSortable={{sortableBool}}">
+              <Hds::AdvancedTable
+                @isSelectable={{selectableBool}}
+                {{! @glint-expect-error - will be fixed by https://hashicorp.atlassian.net/browse/HDS-5090}}
+                @model={{this.model.userDataShort}}
+                @columns={{array
+                  (hash key="id" label="ID" isSortable=sortableBool)
+                  (hash key="name" label="Name")
+                  (hash key="email" label="Email")
+                  (hash key="role" label="Role")
+                }}
+                @hasPinnableFirstColumn={{true}}
```

Comment:
Thought: I'm not super sure about this API... since we already have `@hasStickyFirstColumn`. Say if the first column is pinned by default, you'd need to set `@hasPinnableFirstColumn` and `@hasStickyFirstColumn`?

It is odd for us to mix the terminology of "pinning" and "sticky". Could we simplify to just using `@hasStickyFirstColumn` and if they set it to false it is "pinnable" and if it is undefined it isn't? Just throwing an idea out there.